docs/source/usage.rst

```diff
@@ -233,3 +233,75 @@ Settings related to the S3 bucket
       "aws_region": "eu-central-1",
       "bucket_name": "amo-blocked-pages"
     }
+
+How to create the ``validate_signature`` function on Amazon Lambda?
+-------------------------------------------------------------------
+
+1. `Log into the Amazon Web Service console. <https://console.aws.amazon.com/lambda/home>`_
+2. Click the **Create a Lambda Function** button
+3. Skip the blueprint
+4. Configure the function: **name**, **description**
+5. Select **Python 2.7** runtime
+6. Select **Upload a file from Amazon S3**
+7. Enter your S3 ``lambda.zip`` URL
+8. Use **lambda.validate_signature** handler
+9. Set a **Lambda S3 Exec Role**
+10. Configure a 5 minutes timeout
+11. You can proceed without a VPC
+
+.. image:: images/validate_signature_configuration.png
+    :align: center
+
+How to run the ``validate_signature`` function on Amazon Lambda?
+----------------------------------------------------------------
+
+Once your function has been configured, you can run it.
+
+To do that, select the function and in the **Actions** menu select
+**Configure Test event**.
+
+You will then be able to enter a **Input test event** as a JSON.
+
+.. image:: images/validate_signature_test_event.png
+    :align: center
```

Comment:
Why have an image of JSON that we have in copy/pasteable text below? Isn't this just duplication? (Except that it's a little different for some reason?)